components/docs/plugins/CodeBlock.tsx

```diff
@@ -32,9 +32,9 @@ const PluginDocsCodeBlock = ({ cli, code, language }: CodeBlockProps) => {
       {({ tokens, getLineProps, getTokenProps }) => (
         <pre className={`language-${language}`}>
           {tokens.map((line, i) => (
-            <div {...getLineProps({ line, key: i })}>
+            <div {...getLineProps({ line, key: i })} key={i}>
               {line.map((token, key) => (
-                <span {...getTokenProps({ token, key })} />
+                <span {...getTokenProps({ token, key })} key={key} />
```

Comment:
These don't look right any more
![Screen Shot 2021-12-09 at 4 34 59 PM](https://user-images.githubusercontent.com/303226/145497351-298d0467-3aeb-44b9-8ba9-82366bf622fb.png)

Reply:
Sass importing css can get funky, but should be fixed now.